doc/introduction.rst

```diff
@@ -10,501 +10,327 @@ Introduction
 
-   * :ref:`sphx_glr_auto_examples_00_tutorials_plot_3d_and_4d_niimg.py`
+.. only:: html
 
-   More tutorials can be found :ref:`here <tutorial_examples>`
+  .. image:: /auto_examples/00_tutorials/images/thumb/sphx_glr_plot_single_subject_single_run_thumb.png
+    :alt: Intro to GLM Analysis: a single-session, single-subject fMRI dataset
 
-____
+  :ref:`sphx_glr_auto_examples_00_tutorials_plot_single_subject_single_run.py`
 
-Now, if you want out-of-the-box methods to process neuroimaging data, jump
-directly to the section you need:
+.. raw:: html
 
-* :ref:`decoding`
+      <div class="sphx-glr-thumbnail-title">Intro to GLM Analysis: a single-session, single-subject fMRI dataset</div>
+    </div>
 
-* :ref:`functional_connectivity`
 
-|
+.. raw:: html
 
-Scientific computing with Python
----------------------------------
+    <div class="sphx-glr-thumbcontainer" tooltip="In this example, we will project a 3D statistical map onto a cortical mesh using vol_to_surf, d...">
 
-In case you plan to become a casual nilearn user, note that you will not need
-to deal with number and array manipulation directly in Python.
-However, if you plan to go beyond that, here are a few pointers.
+.. only:: html
 
-Basic numerics
-...............
+  .. image:: /auto_examples/01_plotting/images/thumb/sphx_glr_plot_3d_map_to_surface_projection_thumb.png
+    :alt: Making a surface plot of a 3D statistical map
 
-:Numerical arrays:
+  :ref:`sphx_glr_auto_examples_01_plotting_plot_3d_map_to_surface_projection.py`
 
-  The numerical data (e.g. matrices) are stored in numpy arrays:
+.. raw:: html
 
-  ::
+      <div class="sphx-glr-thumbnail-title">Making a surface plot of a 3D statistical map</div>
+    </div>
 
-    >>> import numpy as np
-    >>> t = np.linspace(1, 10, 2000)  # 2000 points between 1 and 10
-    >>> t
-    array([  1.        ,   1.00450225,   1.0090045 , ...,   9.9909955 ,
-             9.99549775,  10.        ])
-    >>> t / 2
-    array([ 0.5       ,  0.50225113,  0.50450225, ...,  4.99549775,
-            4.99774887,  5.        ])
-    >>> np.cos(t) # Operations on arrays are defined in the numpy module
-    array([ 0.54030231,  0.53650833,  0.53270348, ..., -0.84393609,
-           -0.84151234, -0.83907153])
-    >>> t[:3] # In Python indexing is done with [] and starts at zero
-    array([ 1.        ,  1.00450225,  1.0090045 ])
 
-  `More documentation ...
-  <http://scipy-lectures.github.io/intro/numpy/index.html>`__
+.. raw:: html
 
-  .. warning:: the default integer type used by Numpy is (signed) 64-bit. Several
-              popular neuroimaging tools do not handle int64 Nifti images, so if
-              you build Nifti images directly from Numpy arrays it is recommended
-              to specify a smaller integer type, for example::
+    <div class="sphx-glr-thumbcontainer" tooltip="This example shows manual steps to create and further modify an ROI spatial mask. They represen...">
```

Comment:
unended sentence.

Reply:
This is because that's what the tooltip looks like for these examples in the examples gallery :)